ibc-core/ics26-routing/types/src/error.rs

```diff
@@ -1,16 +1,17 @@
 use displaydoc::Display;
+use ibc_core_host_types::error::DecodingError;
 use ibc_core_host_types::identifiers::PortId;
 use ibc_primitives::prelude::*;
 
 /// Error type for the router module.
-#[derive(Debug, Display)]
+#[derive(Debug, Display, derive_more::From)]
 pub enum RouterError {
-    /// malformed message that could not be decoded: `{description}`
-    MalformedMessageBytes { description: String },
+    /// decoding error: `{0}`
+    Decoding(DecodingError),
```

Comment:
This `Decoding` can be removed now.

Reply:
[7ed66da](https://github.com/cosmos/ibc-rs/pull/1325/commits/7ed66da9fba895d54dd1b0a3ae3c40e0d0e3e998)